pages/index.js

```diff
@@ -453,24 +452,21 @@ const Page = () => (
             />
           </Feature>
           <Feature
-            name="Workshop Bounty Program"
+            name="A comprehensive leader guide"
```

Comment:
I don't know about the word guide... that might connote something different for others

Reply:
hmm

Reply:
Take inspiration from the pros? lol no that's too wordy

Reply:
Ace the First Meeting?

Reply:
Learn from the pros? hmm but I dont want it to sound exclusive

Reply:
Learn from fellow leaders

Reply:
Get Inspired